src/reconcile.py

```diff
@@ -177,6 +195,14 @@ def _local_and_server(
                 content=local_content,
             ),
         )
+
+    try:
+        base_content = clients.repository.get_file_content(
+            path=str(path_info.local_path.relative_to(base_path)),
+            branch=user_inputs.base_branch,
+        )
+    except exceptions.RepositoryClientError:
+        base_content = None
```

Comment:
Shouldn't we re-raise this exception instead of settings base_content to None? This might be thrown in multiple scenarios, including network issues as per https://github.com/PyGithub/PyGithub/blob/master/github/GithubException.py

Reply:
Ah yes, this actually needs more granular exception handling, thanks for pointing that out. Should be fixed on next push.